Tests/SentryTests/Integrations/Session/SentrySessionTrackerTests.swift

```diff
@@ -462,12 +468,17 @@ class SentrySessionTrackerTests: XCTestCase {
     
     private func assertSessionInitSent(sessionStarted: Date) {
         if let session = fixture.client.captureSessionInvocations.last {
-            XCTAssertTrue(session.flagInit?.boolValue ?? false)
```

Comment:
This was particularly suspect, if `flagInit` is ever nil, then this would coalesce to `XCTAssertTrue(false)`.

Reply:
Why is that suspect? If flagInit is nil, then the test would fail as expected, right? Personally I would add an `XCTAssertNotNil(session.flagInit)` before this test so you get an automatic nice error when it is nil, and let the existing test stand as-is.

Reply:
The intent wasn’t clear to me, I think what i changed it to is clearer.

Reply:
``` swift
if let flagInit = session.flagInit {
    XCTAssertTrue(flagInit.boolValue)
} else {
    XCTFail("Expected a session flagInit but there was none.")
}
```

versus

``` swift
XCTAssertNotNil(session.flagInit)
XCTAssertTrue(session.flagInit?.boolValue ?? false)
```

I like the second version over having this if else logic inside of a test, but it's just a minor personal taste thing. I'm good with your version as well.

Reply:
Eh since this PR doesn't really add any value, I'm fine leaving this since it does work.